examples/finite_prime_field.rs

```diff
@@ -0,0 +1,256 @@
+use noether::{
+    AssociativeAddition, AssociativeMultiplication, CommutativeAddition, CommutativeMultiplication,
+    DistributiveAddition, DistributiveMultiplication, FiniteField,
+};
+use num_traits::{Euclid, Inv, One, Zero};
+use std::ops::{Add, AddAssign, Div, DivAssign, Mul, MulAssign, Neg, Rem, Sub, SubAssign};
+
+/// The trait hierarchy in Noether is useful for verifying the correct and idiomatic implementation
+/// of algebraic structures in rust.
+///
+/// We take here, as an example, the implementation of a finite field.
+
+#[derive(Clone, Copy, Debug)]
+pub struct FinitePrimeField<const L: usize, const D: usize> {
+    modulus: [u64; L],
+    _value: [u64; L],
```

Comment:
Naively, what are L,D,modulus,_value?

Reply:
Length in words, double length, then the modulus for the field, admittedly this could use a comment.